web/app/utils/email.js

```diff
@@ -0,0 +1,31 @@
+import store from '../common/store';
+import { siteId } from '../common/settings';
+import { deleteMe } from '../common/api';
+
+export function getDeleteInformationMessage(userId, siteId, link) {
+  const subject = encodeURIComponent("Request to delete user's information");
+  const message = encodeURIComponent(`Request to delete all information about ${userId} from remark42 on ${siteId}
+
+[you can provide the reason for removal request, optional]
+
+=== DO NOT REMOVE THE TEXT BELOW THIS LINE ===
+
+site: ${siteId}
+user: ${userId}
+link: ${link}
+`).replace('%0A', '%0D%0A');
```

Comment:
Can you explain this replacement, please? I think you need to add comment about it.